src/algebra/group/units.lean

```diff
@@ -110,8 +110,8 @@ by rw [←units.coe_one, eq_iff]
 
 @[to_additive] lemma val_coe : (↑a : α) = a.val := rfl
 
-@[norm_cast, to_additive] lemma coe_inv : ((a⁻¹ : units α) : α) = a.inv := rfl
-attribute [norm_cast] add_units.coe_neg
+@[norm_cast, to_additive] lemma coe_inv'' : ((a⁻¹ : units α) : α) = a.inv := rfl
```

Comment:
I assume the single-primed name is already taken?

Reply:
Yes, for `group_with_zero`.